packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -16,78 +16,97 @@ export const TokenNamesAndLogos: Record<string, string> = {
   ETH: "./logos/ethereum-eth-logo.svg",
 };
 
+export const TokenAndCoinGeckoIds: Record<string, string> = {
+  FTM: "fantom",
+  BUSD: "binance-usd",
+  DAI: "dai",
+  ETH: "ethereum",
+};
+
```

Comment:
There's a lot of duplication between this file and the `utils.ts` file under `round-manager`. Should we consolidate both under the `common` package?

Happy to create an issue (if we don't have one yet) and go through our repo to do some consolidation across rm, re and builder if we think this is a good idea.